frontend/views/containers/contributions/ContributionItem.vue

```diff
@@ -5,37 +5,60 @@
   div(v-if='hasWhoElse')
     transition(name='replace-list')
       div(
-        v-if='isVisible'
+        v-if='ephemeral.isVisible'
         key='visible'
       )
         .c-contribution-list(v-safe-html='listOfName')
 
         i18n.is-unstyled.is-link-inherit.link(
           tag='button'
           type='button'
-          @click='isVisible = !isVisible'
+          @click='toggleVisibility'
         ) Hide
 
       div(
         v-else
         key='hidden'
-        @click='isVisible = !isVisible'
+        @click='toggleVisibility'
         v-safe-html='contributionText'
       )
 
   .c-contribution-list(
-    v-else=''
-    v-safe-html='contributionText'
+    v-else
+    v-safe-html:button='contributionText'
+    @click='showProfileTooltip'
   )
+
+  .c-profile-card-container(
+    v-if='ephemeral.profileTooltip.show'
+    v-on-clickaway='closeProfileTooltip'
+  )
+    .c-profile-card-overlay(
+      @click.stop='closeProfileTooltip'
+    )
+
+    profile-card-content.c-card(
+      :contractID='ephemeral.profileTooltip.userId'
+      :on-post-cta-click='closeProfileTooltip'
+      @modal-close='closeProfileTooltip'
+    )
```

Comment:
Why is this using `ProfileCardContent` directly instead of `ProfileCard`? Could you add a comment explaining?